src/agent/src/cdh.rs

```diff
@@ -90,6 +80,55 @@ impl CDHClient {
     }
 }
 
+pub async fn init_cdh_client() -> Result<()> {
+    let mut cdh_client = CDH_CLIENT.write().await;
+    if cdh_client.is_none() {
+        let client = CDHClient::new().context("Failed to create CDH Client")?;
+        *cdh_client = Some(client);
+    }
+    Ok(())
+}
```

Comment:
How hard would it be to avoid using an `Arc<RwLock>` here? I don't know enough about Rust but what's the alternative? Dependency injection with cloning? Explicit lifetimes? Seems unfortunate to revert to run-time checks.

Reply:
The `CDH_CLIENT` is essentially a global static reference, requiring a synchronization primitive to ensure thread safety in the rust. While in the usage of cdh in kata, the `CDH_CLIENT` is initialized once during the agent’s startup and subsequently read multiple times for handling RPC requests. So I think I can explore an alternative approach to avoid using RwLock or Mutex.

Reply:
I updated the code to use `OnceCell` instead of `RwLock` or `Mutex` to avoid unnecessary locking.